RATapi/project.py

```diff
@@ -27,9 +27,11 @@
 from RATapi.utils.enums import Calculations, Geometries, LayerModels, Priors, TypeOptions
 
 
+# note for these discriminators that the before-validator discriminate_ambiguous_dicts
+# guarantees we don't run into the ambiguous case of a sequence of dicts
 def discriminate_layers(layer_input):
     """Union discriminator for layers."""
-    if isinstance(layer_input, (RATapi.ClassList, list)):
+    if isinstance(layer_input, (RATapi.ClassList, collections.abc.Sequence)):
```

Comment:
I think a ClassList satisfies `isinstance(layer_input, collections.abc.Sequence)`. If so, you can remove `RATapi.ClassList` from the condition.